dcos/dcos-ui
# DCOS-43324: fix Cosmos icons

The PR drops parsing of cosmos URL as that can break URL on some clusters.

## Testing

1. add `--https` to npm start like so `"start": "NODE_ENV=development webpack-dev-server --https --config ./webpack/webpack.dev.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",`
2. Launch any catalog service with an icon
3. Verify that icon is visible

## Trade-offs

Dev mode without `https` won't show icons
We can' switch to https on dev just now as webpack crashes on the catalog page
